scripts/sharedServices/aws/sqs/sendSqsMessage.ts

```diff
@@ -0,0 +1,35 @@
+import {
```

Comment:
As discussed, let's not copy and paste something that's nearly identical here, and just set an environment variable in code to let us use the version of the same thing in `src`